docs/semgrep-assistant/overview.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: overview
+append_help_link: true
+title: AI-assisted triage and autofix
+hide_title: true
+description: Learn how Semgrep Assistant can provide recommendations for triage and remediation of Semgrep findings. 
+tags:
+  - Semgrep Code
+  - Semgrep Supply Chain
+  - Semgrep Secrets
+  - Semgrep Cloud Platform
+  - Semgrep Assistant
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Semgrep Assistant overview
+
+Semgrep Assistant provides GPT-4-powered security recommendations to help you review, triage, and remediate your Semgrep findings.
+
+![Screenshot of Semgrep Assistant message in GitHub](/img/semgrep-assistant-github.png)
+*Figure*. Semgrep Assistant detects a false positive.
+
+## Support and availability
+
+Semgrep Assistant supports findings generated by Semgrep Code, Semgrep Supply Chain, and Semgrep Secrets, and, in many cases, requires using Semgrep Cloud Platform. It is not available on the Semgrep CLI.
+
+Semgrep Assistant is available to users of the following source code managers (SCMs):
+  * GitHub Cloud
+  * GitLab, including SaaS and self-managed plans
```

Comment:
I think GitLab has a self-managed plan that is not self-hosted, right? Is that a pertinent distinction here?